src/Migrations.php

```diff
@@ -235,8 +236,14 @@ public function seed($options = [])
     {
         $this->setCommand('seed');
         $input = $this->getInput('Seed', [], $options);
-        $params = ['default', $input->getOption('seed')];
-        $this->run('Seed', $params, $input);
+
+        $seed = $input->getOption('seed');
+        if (empty($seed)) {
```

Comment:
Ideally, we dont use cloaking where not necessary, so !$seed would be cleaner here.